### PR TITLE
Issue155 csv header

### PR DESCRIPTION
**Pull request by dhblum**
This is for https://github.com/ibpsa/project1-boptest/issues/155.

- Uses ``pd.read_csv(comment='#')`` within ``data.data_manager.Data_Manager``
- Adds example header information to ``testcases/testcase1/models/Resources/*.csv``
- Updates requirements documentation with use of ``#`` for header information in data csv files

### CODE REVIEW

**Comment by dhblum** [comment]
@Mathadon Can you review this proposed implementation of adding header information to data csv files?

File: testcases/testcase1/models/Resources/emissions.csv [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Optional header information
+# time: time of emissions factor [s]
+# EmissionsElectricPower: Emissions factor of electric poweer [kgCO2-e/kWh]
```
Comment (Mathadon) [inline testcases/testcase1/models/Resources/emissions.csv]: I'm not familiar with the `-e` notation. Does e stand for emission?

Comment (dhblum) [reply testcases/testcase1/models/Resources/emissions.csv]: CO2 equivalent.

Comment (Mathadon) [reply testcases/testcase1/models/Resources/emissions.csv]: okay!

**Comment by Mathadon** [comment]
this can be merged

**Pull request merged**